cabal-install/Distribution/Client/ProjectOrchestration.hs

```diff
@@ -483,6 +484,14 @@ printPlan verbosity
     showFlagValue (f, False)  = '-' : showFlagName f
     showFlagName (PD.FlagName f) = f
 
+    showProfiling elab =
+        (if elabProfLib elab
+            then " profile"
+            else "") ++
+        (if elabProfExe elab
+            then " exe-profile"
+            else "")
```

Comment:
Okay, now I understand.  I found it confusing at first, especially when it listed "exe-profile" for a library-only package.  Would it make sense to just list "profile" for either flag, or only show the profiling information at a higher verbosity?